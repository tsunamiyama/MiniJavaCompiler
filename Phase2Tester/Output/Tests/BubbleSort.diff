--- SelfTestCases/BubbleSort.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/BubbleSort.out	2021-02-13 18:30:50.120287100 -0800
@@ -1,22 +1,5 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+--- Error (ProgramFault) ---
+use of uinitialized value
+  Main() line 18
+  BBS.Start(<Heap.0>+0 10) line 32
+  BBS.Init(<Heap.0>+0 10) line 290
