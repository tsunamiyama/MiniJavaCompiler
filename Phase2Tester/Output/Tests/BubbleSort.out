--- Error (ProgramFault) ---
use of uinitialized value
  Main() line 18
  BBS.Start(<Heap.0>+0 10) line 32
  BBS.Init(<Heap.0>+0 10) line 290
